ecs-cli/modules/config/readwriter_v1.go

```diff
@@ -90,21 +116,19 @@ func readClusterConfig(path string, clusterConfigKey string, cliConfig *CLIConfi
 	cliConfig.Region = cluster.Region
 	cliConfig.Cluster = cluster.Cluster
 	cliConfig.ComposeServiceNamePrefix = cluster.ComposeServiceNamePrefix
+	// set prefixes to empty
+	cliConfig.CFNStackNamePrefix = ""
+	cliConfig.ComposeProjectNamePrefix = ""
```

Comment:
We no longer accept `CFNStackNamePrefix` and `ComposeProjectNamePrefix`. Why do you have to set them to empty explicitly?